src/backend/marsha/core/serializers/video.py

```diff
@@ -466,6 +474,19 @@ class Meta:  # noqa
     live_info = serializers.SerializerMethodField()
     xmpp = serializers.SerializerMethodField()
 
+    def to_representation(self, instance):
+        """Object instance -> Dict of primitive datatypes.
+
+        Force a default value to live_type field if there is no value and live_state
+        is not null.
+        """
+        video_representation = super().to_representation(instance)
+
+        if instance.live_state is not None and instance.live_type is None:
+            video_representation["live_type"] = RAW
+
+        return video_representation
```

Comment:
I don't like this version because it doesn't solve the problem when using the video model instance in the code.